Ladybird/Qt/TabBar.cpp

```diff
@@ -45,6 +45,39 @@ void TabBar::contextMenuEvent(QContextMenuEvent* event)
         tab->context_menu()->exec(event->globalPos());
 }
 
+void TabBar::mousePressEvent(QMouseEvent* event)
+{
+    event->ignore();
+
+    auto rect_of_current_tab = tabRect(tabAt(event->pos()));
+    m_position_in_selected_tab_while_dragging = QPoint(event->pos().x() - rect_of_current_tab.x(), event->pos().x() - rect_of_current_tab.x());
```

Comment:
The x co-ordinate is being repeated here. Could you store just the X position since that's all we're interested in?

```c++
m_x_position_in_selected_tab_while_dragging = event->pos().x() - rect_of_current_tab.x();
```